homeassistant/components/huawei_lte/sensor.py

```diff
@@ -109,6 +116,8 @@ def setup_platform(
     data = hass.data[DATA_KEY].get_data(config)
     sensors = []
     for path in config.get(CONF_MONITORED_CONDITIONS):
+        if path == "traffic_statistics":  # backwards compatibility
```

Comment:
Please make sure that this is conform to https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Reply:
I will in a later set of changes. This is stuff I need to get out of the way before that, see https://github.com/home-assistant/home-assistant.io/pull/9943 for more explanation. (But just quickly here too, the ADR doesn't actually quite apply, because it says "fetched in a single API request" -- in huawei_lte's case the data is not available in a single http request, every category requires its own.)

Reply:
I'd like to also note that it would be very good to get these changes out of the way of the more intrusive future ones (config overhaul, unique_id changes, etc) -- much easier to review that way.